.github/actions/add-jira-issue-code-to-pull-request/AddJiraIssueCodeToPullRequest.ps1

```diff
@@ -13,30 +12,36 @@ $jiraBrowseUrl = $JiraBaseUrl + '/browse/';
 $originalTitle = $Title
 $originalBody = $Body
 
-if ($Branch -NotLike "*issue*") {
+if ($Branch -NotLike "*issue*")
+{
     Exit
 }
 
 $Branch -match '(\w+-\d+)'
 $issueKey = $matches[0]
 $issueLink = "[$issueKey]($jiraBrowseUrl$issuekey)"
 
-if ($Title -NotLike "*$issueKey*") {
+if ($Title -NotLike "*$issueKey*")
+{
     $Title = $issueKey + ": " + $Title
 }
 
-if (-Not $Body) {
+if (-Not $Body)
+{
     $Body = $issueLink
 }
-elseif ($Body -NotLike "*$issueKey*") {
+elseif ($Body -NotLike "*$issueKey*")
+{
     $Body = $issueLink + "`n" + $Body
 }
-elseif ($Body -NotLike "*``[$issueKey``]``($jiraBrowseUrl$issuekey``)*") {
+elseif ($Body -NotLike "*``[$issueKey``]``($jiraBrowseUrl$issuekey``)*")
+{
     $Body = $Body.replace($issueKey, $issueLink)
 }
 
-if (($Title -ne $originalTitle) -or ($Body -ne $originalBody)) {
-    $bodyParams = @{"title" = $Title; "body" = $Body} | ConvertTo-Json
+if (($Title -ne $originalTitle) -or ($Body -ne $originalBody))
+{
+    $bodyParams = @{"title" = $Title; "body" = $Body } | ConvertTo-Json
     $url = "https://api.github.com/repos/$GitHubRepository/pulls/$PullRequestId"
     Invoke-WebRequest $url -Headers (Get-GitHubApiAuthorizationHeader) -Method Patch -Body $bodyParams
 }
```

Comment:
If I remove the issue link from the PR description and re-run the corresponding workflow job, was it supposed to re-add it again or only when the PR is being created?

Reply:
While this is a nice side effect, I don't think we intended this to actually run on all pushes, it just happens to do without any further checks. But an `if` on the GH even to see if it's a PR `opened` event would be great.

Reply:
OK, I don't think that's a problem overall, but let's not add that to this issue, because there's already a lot of scope creep.

Reply:
Added https://github.com/Lombiq/GitHub-Actions/issues/152 for this.